docs/install_guides/installing-openvino-apt.md

```diff
@@ -159,7 +159,11 @@ apt list --installed | grep openvino
 
 After you have installed OpenVINO Runtime, if you decided to [install OpenVINO Model Development Tools](installing-model-dev-tools.md), make sure that you install external software dependencies first.
 
-Refer to <a href="openvino_docs_install_guides_installing_openvino_linux.html#install-external-dependencies">Install External Software Dependencies</a> for detailed steps.
+OpenVINO provides a script in the installation directory. Run the following commands:
+```sh
+cd <INSTALL_DIR>/install_dependencies
+sudo -E ./install_openvino_dependencies.sh
```

Comment:
we don't need to install additional dependencies for apt / rpm, because such packages contain meta information which helps to install dependencies together with OpenVINO. I.e. installing `apt-get install openvino` installs all such dependencies from `./install_openvino_dependencies.sh`

Reply:
Then can we remove this ### Step 3 (Optional): Install Software Dependencies section? Is it the same for YUM installation?